frappe/__init__.py

```diff
@@ -1021,15 +1021,19 @@ def read_file(path, raise_not_found=False):
 	else:
 		return None
 
-def get_attr(method_string):
+def get_attr(method_string, args=[]):
```

Comment:
Thanks, this looks a lot better. Although, [apparently mutables are not a good default kwargs](https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/). Can we instantiate everything to None instead?

Reply:
Sure will make the changes thanks.